api/v1beta2/awscluster_types.go

```diff
@@ -208,6 +208,11 @@ type AWSLoadBalancerSpec struct {
 	// +optional
 	AdditionalSecurityGroups []string `json:"additionalSecurityGroups,omitempty"`
 
+	// AdditionalListeners sets the additional listeners for the control plane load balancer.
+	// This is only applicable to Network Load Balancer (NLB) types for the time being.
+	// +optional
+	AdditionalListeners []*AdditionalListenerSpec `json:"additionalListeners,omitempty"`
```

Comment:
Does this list need to be a map/set semantic at all by openapi standards? Eg must the port be unique in each item in this list? If so `+listType=map` and `+listMapKey=port` would be useful to prevent duplicates